hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/configuration/HBaseToCloudBigtableReplicationConfiguration.java

```diff
@@ -59,7 +59,7 @@ private HBaseToCloudBigtableReplicationConfiguration() {}
   // batch capacity by counts and bytes. These capacity are set by `replication.source.nb.capacity`
   // and `replication.source.size.capacity` config keys.
   public static final long DEFAULT_BATCH_SIZE_IN_BYTES = 500_000;
-  // TODO: Plumb this config key to HBase1.x
+  // TODO: Use this config key for bigtable peer rather than REPLICATION_WALENTRYFILTER_CONFIG_KEY.
   // Allow for config-driven WALEntryFilters for any bigtable replication peer
   public static final String BIGTABLE_REPLICATION_WALENTRYFILTER_CONFIG_KEY =
       "google.bigtable.replication..source.custom.walentryfilters";
```

Comment:
This config should not even exist now, does it?

Reply:
We can delete this for now as it is not being used anywhere.